kratos/processes/structured_mesh_generator_process.cpp

```diff
@@ -263,7 +263,7 @@ namespace Kratos
             cell_points.push_back(coordinates);
         }
 
-        constexpr triangle_connectivity_in_cell_type connectivity_cases[number_of_cases] = { {{ 1,3,2 }},{{ 1,4,3 }} };
+        constexpr triangle_connectivity_in_cell_type connectivity_cases[number_of_cases] = { {{ 0,2,1 }},{{ 0,3,2 }} };
```

Comment:
make sense to me however...don't you have too many braces??

also, does this compile on win?

Reply:
i am telling since some time back we had a segfault at compilation time with a constexpr of this type

Reply:
just adding that the segfault was in window. @roigcarlo am i remembering correctly?

Reply:
Well, if I am not mistaken, I think the triple braces should be there: the outermost pair are the array's braces. Then,  each of the double braces are formed by an outer pair corresponding to the brace initialization of the std::array, where you are feeding them arrays that are being created on the spot (hence the innermost pair of braces).

Check this out: [braced initialization](https://stackoverflow.com/questions/11734861/when-can-outer-braces-be-omitted-in-an-initializer-list)